### PR TITLE
Npcs rebalance part01  #868

### PR DESCRIPTION
**Pull request by XxSARGExX**
NPC's completely overhauled

- New Npc loadouts
- All NPC spawns in All Zones adjusted to the new level system
- Adjusted some pilots
- Adjusted Random Mission level
- Added/Adjusted Mission Vignette in Magellan to have more Lane Hacker

### CODE REVIEW

File: DATA/MISSIONS/faction_prop.ini [hunk 1]
```diff
@@ -73,26 +91,17 @@ legality = lawful
...
-npc_ship = li_n_li_elite_d16-18
-npc_ship = li_p_li_fighter_d16-18
-npc_ship = li_n_li_elite2_d16-18
-npc_ship = li_n_li_elite_d19
-npc_ship = li_n_li_elite2_d19
-npc_ship = li_p_li_fighter_d19
-; npc_ship = li_p_li_fighter_boss01
-; npc_ship = li_n_li_elite2_boss01
-; npc_ship = li_n_li_elite_boss01_esc
+npc_ship = li_lsfe_li_fighter_d1_a
```
Comment (erikroe) [inline DATA/MISSIONS/faction_prop.ini:94]: lsfe instead of lsf

File: DATA/MISSIONS/faction_prop.ini [hunk 2]
```diff
@@ -561,52 +633,51 @@ legality = lawful
...
-npc_ship = npc_co_kt_fighter4_d1-2
-npc_ship = npc_co_kt_fighter_d3-4
-npc_ship = npc_co_kt_fighter2_d3-4
-npc_ship = npc_co_kt_fighter3_d3-4
-npc_ship = npc_co_kt_fighter4_d3-4
-npc_ship = npc_co_kt_fighter_d5-6
-npc_ship = npc_co_kt_fighter2_d5-6
-npc_ship = npc_co_kt_fighter3_d5-6
-npc_ship = npc_co_kt_fighter4_d5-6
+npc_ship = npc_co_kt_fighter_d1_a
```
Comment (erikroe) [inline DATA/MISSIONS/faction_prop.ini:636]: Why `npc_` where other loadouts do not have this `npc_` prefix?

File: DATA/MISSIONS/npcships.ini [hunk 3]
```diff
@@ -54,6 +144,51 @@ state_graph = TRANSPORT
 pilot = transport_easy
 npc_class = lawful, class_transport, transport_large, d1, d2, d3, d4, d5, d6, d7, d8, d9, d10, d11, d12, d13, d14, d15, d16, d17, d18, d19
 
+[NPCShipArch]
+nickname = npc_co_br_large_transport_d1-19
+level = d10
+loadout = co_br_large_transport_loadout_d1-19
+ship_archetype = cv_ship_transport03_npc
+state_graph = TRANSPORT
+pilot = transport_easy
```
Comment (erikroe) [inline DATA/MISSIONS/npcships.ini:153]: I wonder if we shouldn't give transport_ace to all the transports. Small, med, large, etc.

File: DATA/MISSIONS/npcships.ini [hunk 4]
```diff
@@ -120,656 +390,761 @@ npc_class = lawful, class_cruiser, d1, d2, d3, d4, d5, d6, d7, d8, d9, d10, d11,
 ;; ZIVIL Starflyer
 
 [NPCShipArch]
-nickname = npc_co_ge_fighter_d1-2
+nickname = npc_co_ge_fighter_d1_a
+;;nickname = npc_co_ge_fighter_d1-2
```
Comment (erikroe) [inline DATA/MISSIONS/npcships.ini:394]: Is there a reason to keep those old names as comment?

File: DATA/MISSIONS/npcships.ini [hunk 5]
```diff
@@ -120,656 +390,761 @@ npc_class = lawful, class_cruiser, d1, d2, d3, d4, d5, d6, d7, d8, d9, d10, d11,
...
 
-[NPCShipArch]
-nickname = npc_co_ge_fighter_d11-12
-level = d11
-loadout = co_ge_fighter_loadout_d11-12
-ship_archetype = cv_ship_fighter_verylight01_npc
-state_graph = FIGHTER
-pilot = pilot_company_hard
-npc_class = lawful, class_light, class_fighter, class_mission, d11, d12
+;;[NPCShipArch]
```
Comment (erikroe) [inline DATA/MISSIONS/npcships.ini:442]: Are those commented sections needed for something?

File: DATA/MISSIONS/npcships.ini [hunk 6]
```diff
@@ -866,7024 +1241,9022 @@ npc_class = lawful, freighter
...
-nickname = fc_m_pi_freighter_d9-19
-loadout = fc_m_pi_freighter_loadout02
-level = d20
-ship_archetype = pi_ship_freighter01_npc
-pilot = freighter_ace
+nickname = gd_z_fighter5_d2_a
+level = d2
+loadout = gd_z_fighter5_loadout_d2_a
+ship_archetype = cv_ship_fighter_medium01_npc
 state_graph =
```
Comment (erikroe) [inline DATA/MISSIONS/npcships.ini:9675]: Are Bosses used?

**Review by erikroe — CHANGES_REQUESTED** [review]

File: DATA/MISSIONS/npcships.ini (hunk 5 above)
Comment (XxSARGExX) [reply DATA/MISSIONS/npcships.ini:442]: These are old (not used anymore) entries, i used them to compare with old and new Npc Balancing, i will create an extra branch for "clean up"

File: DATA/MISSIONS/npcships.ini (hunk 4 above)
Comment (XxSARGExX) [reply DATA/MISSIONS/npcships.ini:394]: These are old (not used anymore) entries, i used them to compare with old and new Npc Balancing, i will create an extra branch for "clean up"

File: DATA/MISSIONS/faction_prop.ini (hunk 2 above)
Comment (XxSARGExX) [reply DATA/MISSIONS/faction_prop.ini:636]: will be changed in future NPC Adjustment phases

File: DATA/MISSIONS/npcships.ini (hunk 3 above)
Comment (XxSARGExX) [reply DATA/MISSIONS/npcships.ini:153]: will be changed in future NPC Adjustment phases

File: DATA/MISSIONS/npcships.ini (hunk 6 above)
Comment (XxSARGExX) [reply DATA/MISSIONS/npcships.ini:9675]: currently not used (placeholder) but i will use them in future NPC Adjustment phases

**Review by erikroe — APPROVED** [review]

**Review by erikroe — APPROVED** [review]

**Pull request merged**